packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -116,37 +139,21 @@ const CompleteResetPassword = ({
       }
     };
 
-    const handleRecoveryKeyStatus = async () => {
-      if (!location.state?.lostRecoveryKey) {
-        await checkForRecoveryKeyAndNavigate(params.email);
-      }
+    const loadCompleteResetPassword = () => {
```

Comment:
Tiny nit but maybe `render` instead of `load` to match other naming conventions in the project?